.github/workflows/e2e.yml

```diff
@@ -12,15 +12,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        cable_driver: ['libreswan', 'wireguard']
         deploytool: ['operator']
         globalnet: ['', 'globalnet']
-        cable_driver: ['libreswan', 'wireguard']
+        k8s_version: ['1.17.17']
```

Comment:
You're not actually using this, so all the jobs will actually run with whatever is default..

Reply:
Good catch, thanks @mkolesnik. I double-checked and I didn't make this error elsewhere:

https://github.com/submariner-io/submariner-operator/pull/1183/files
https://github.com/submariner-io/lighthouse/pull/495/files
https://github.com/submariner-io/submariner-charts/pull/130/files

Reply:
Double-checked here and it's pulling the correct node images now:

> 2021-04-07T15:07:20.3318828Z [cluster3]  ✓ Ensuring node image (kindest/node:v1.19.7)

Thanks again MIke, great catch.